docs/src/content/docs/fr/guides/overriding-components.md

```diff
@@ -0,0 +1,133 @@
+   ```
+
+## Réutiliser un composant intégré
+
+Vous pouvez utiliser avec les composants par défaut de Starlight comme vous le feriez avec les vôtres : en les important et en les affichant dans vos propres composants personnalisés. Cela vous permet de conserver toute l'interface utilisateur de base de Starlight dans votre design, tout en ajoutant des éléments supplémentaires à côté.
+
+L'exemple ci-dessous montre un composant personnalisé qui affiche un lien de courriel suivi du composant `SocialLinks` par défaut :
+
+```astro {4,8}
+---
+// src/components/EmailLink.astro
+import type { Props } from '@astrojs/starlight/props';
+import Default from '@astrojs/starlight/components/SocialIcons.astro';
+---
+
+<a href="mailto:houston@example.com">Me contacter par courriel</a>
+<Default {...Astro.props}><slot /></Default>
+```
+
+Lorsque vous affichez un composant intégré dans un composant personnalisé :
+
+- Décomposez `Astro.props` dans celui-ci. Cela permet de s'assurer qu'il reçoit toutes les données dont il a besoin pour être affiché.
+- Ajoutez un élément [`<slot />`](https://docs.astro.build/fr/core-concepts/astro-components/#les-emplacements-slots) à l'intérieur du composant par défaut. Cela permet de s'assurer que si le composant reçoit des éléments enfants, Astro sait où les afficher.
+
+## Utiliser les données de la page
+
+Lorsque vous redéfinissez un composant de Starlight, votre implémentation personnalisée reçoit un objet `Astro.props` standard contenant toutes les données de la page courante.
+Cela vous permet d'utiliser ces valeurs pour contrôler la manière dont votre composant est affiché.
+
+Par exemple, vous pouvez lire les valeurs du frontmatter de la page via `Astro.props.entry.data`. Dans l'exemple suivant, un composant [`PageTitle`](/fr/reference/overrides/#pagetitle) de remplacement utilise ces données pour afficher le titre de la page courante :
+
+```astro {5} "{title}"
+---
+// src/components/Title.astro
+import type { Props } from '@astrojs/starlight/props';
+
+const { title } = Astro.props.entry.data;
+---
+
+<h1 id="_top">{title}</h1>
+
+<style>
+  h1 {
+    font-family: 'Comic Sans';
+  }
+</style>
+```
+
+Pour en savoir plus sur les props disponibles, consultez la [référence des redéfinitions](/fr/reference/overrides/#props-des-composants).
+
+### Redéfinir uniquement sur des pages spécifiques
+
+Les redéfinitions de composants s'appliquent à toutes les pages. Cependant, vous pouvez les utiliser conditionnellement grâce aux valeurs de `Astro.props` pour déterminer quand afficher votre interface utilisateur personnalisée, quand afficher l'interface utilisateur par défaut de Starlight, ou même quand afficher quelque chose de complètement différent.
+
+Dans l'exemple suivant, un composant redéfinissant le composant [`Footer`](/fr/reference/overrides/#footer-1) de Starlight affiche "Construit avec Starlight 🌟" sur la page d'accueil uniquement, et affiche sinon le pied de page par défaut sur toutes les autres pages :
```

Comment:
This link needs to be `#footer` — in English we have two identical headings — “Footer” and “`Footer`” — but in French these are “Pied de page” and “`Footer`”, so this heading doesn’t have the extra number added.

```suggestion
Dans l'exemple suivant, un composant redéfinissant le composant [`Footer`](/fr/reference/overrides/#footer) de Starlight affiche "Construit avec Starlight 🌟" sur la page d'accueil uniquement, et affiche sinon le pied de page par défaut sur toutes les autres pages :
```

Reply:
Damn, excellent catch, thanks a lot for the review 🙌